Source/WebCore/bindings/js/JSDOMPromiseDeferredForward.h

```diff
@@ -64,6 +64,7 @@ template<typename T> struct IDLSequence;
 template<typename T> struct IDLFrozenArray;
 template<typename K, typename V> struct IDLRecord;
 template<typename T> struct IDLPromise;
+template<typename T> struct IDLPromiseIgnoringSuspension;
```

Comment:
These forward declarations don’t need template argument names. We should remove all the `T`, `K`, and `V`.